lib/lita/adapters/slack.rb

```diff
@@ -9,7 +9,9 @@ class Slack < Adapter
       # Required configuration attributes.
       config :token, type: String, required: true
       config :proxy, type: String
-      config :parse, type: String
+      config :parse, type: [String, NilClass]
+      config :link_names, type: [FalseClass, TrueClass, NilClass]
```

Comment:
> only get checked when the value is explicitly assigned, and all config values default to nil.

Yeah, it was basically so we could change it in the specs. I replaced the assignation by a stub. It's good enough i think.